### ISSUE TITLE
[gulp] Order of components listed "breaks" layout

### ISSUE THREAD

**Issue by grandeljay** [fomantic/Fomantic-UI#2368]
# Bug Report
The dropdown input is too long and has a width of `100%` instead of `auto`. The order of the components in `semantic.json` seem to be relevant.

## Steps to reproduce
Use the components `dropdown` and `form` (in this order) in your `semantic.json` and build the files.

## Expected result
The Dropdown input fits the container.

## Actual result
The Dropdown input is larger than the container.

## Testcase
Taken from my `semantic.json`
```json
"components": [
    "api",
    "breadcrumb",
    "button",
    "checkbox",
    "container",
    "dimmer",
    "divider",
    "dropdown",
    "flag",
    "form",
    "grid",
    "header",
    "icon",
    "input",
    "label",
    "list",
    "loader",
    "menu",
    "message",
    "modal",
    "popup",
    "reset",
    "search",
    "segment",
    "site",
    "tab",
    "table",
    "text",
    "toast",
    "transition"
  ],
```

## Screenshot (if possible)
![image](https://user-images.githubusercontent.com/45571053/167560524-50ee88fc-1368-4dab-a726-4efb12f2dde2.png)

Top image is my result, bottom image is from the official fiddle.

Removing the component list or changing the order fixes the issue. Sorting them alphabetically (like they are now) causes this issue.

## Version
2.9.0-beta.279

As I am writing this, I noticed my `semantic.json` mentions `"version": "2.8.8"`, but I have switched to nightly builds since. Do I need to manually change this?

**Comment by jamessampford** [comment]
Using the JSFiddle (https://jsfiddle.net/Lfs5wpkm/1/) it is possible to replicate this as is by wrapping the dropdown in a `.field` where the width is constrained (which is probably more accurately what the screenshot is showing)

![image](https://user-images.githubusercontent.com/1376843/167619804-2452b1fc-2c15-4e5f-acb2-938a8801b5e3.png)

It may be that the fix should be to fix how the input is done (in terms of width it should be 100% but constrained by the parent `.menu` class boundaries) within the dropdown as opposed to changing the ordering - since presumably developers can put the components into any particular order

**Comment by grandeljay** [comment]
> Using the JSFiddle it is possible to replicate this

Oh nice! I was so sure that I had tried that and was not able to reproduce this. Glad you discovered this. I guess that means this is a Dropdown component bug and has nothing to do with gulp.

**Comment by lubber-de** [comment]
Fixed by #2371
